src/ecephys_etl/modules/align_timestamps/_schemas.py

```diff
@@ -0,0 +1,92 @@
+from argschema import ArgSchema, ArgSchemaParser
+from argschema.schemas import DefaultSchema
+from argschema.fields import Nested, InputDir, String, Float, Dict, Int, List
+
+
+class ProbeMappable(DefaultSchema):
+    name = String(
+        required=True,
+        help='What kind of mappable data is this? e.g. "spike_timestamps"',
```

Comment:
Can we specify a finite list of valid values here? See

https://github.com/AllenInstitute/ophys_etl_pipelines/blob/staging/segmentation_dev/src/ophys_etl/modules/segmentation/modules/schemas.py#L88-L89

for an example